src/library_wasmfs_jsimpl.js

```diff
@@ -75,6 +69,7 @@ mergeInto(LibraryManager.library, {
     _emscripten_proxy_finish(ctx);
   },
 
+  _wasmfs_jsimpl_async_write__sig: 'vppppjp',
```

Comment:
```suggestion
  _wasmfs_jsimpl_async_write__sig: 'vpppppjp',
```

Reply:
Thanks!  It seems like this function as no callers yet.. which is a little odd.  @tlively is that expected?